### PR TITLE
Update dice.py

### PR DESCRIPTION
**Pull request by abrahamalbert18**
Whenever you're asking JARVIS to roll a die, it's just replying with unicode character of dice in python. The problem here is, the output character looks so small on larger resolution desktops or laptops or mobiles.

### CODE REVIEW

**Comment by swapagarwal** [comment]
Hi @abrahamalbert18, this nice contribution was needed yesterday! 😜

File: modules/src/dice.py [hunk 1]
```diff
@@ -3,14 +3,12 @@
...
-}
+dice_sides = {1: 'https://upload.wikimedia.org/wikipedia/commons/6/62/Kismet-Ace.png',
+              2: 'https://upload.wikimedia.org/wikipedia/commons/6/6d/Kismet-Deuce.png',
+              3: 'https://upload.wikimedia.org/wikipedia/commons/f/f2/Kismet-Trey.png',
+              4: 'https://upload.wikimedia.org/wikipedia/commons/1/11/Kismet-Four.png',
+              5: 'https://upload.wikimedia.org/wikipedia/commons/8/8e/Kismet-Five.png',
+              6: 'https://upload.wikimedia.org/wikipedia/commons/4/4e/Kismet-Six.png'}
 
 def process(input, entities=None):
     roll = TextTemplate(dice_sides[random.randint(1, 6)]).get_message()
```
Comment (swapagarwal) [inline modules/src/dice.py]: You'd have to use AttachmentTemplate to send images in the response. See https://github.com/swapagarwal/JARVIS-on-Messenger/blob/master/modules/src/coin.py for reference.

File: modules/src/dice.py [hunk 2]
```diff
@@ -3,14 +3,12 @@
...
 
-dice_sides = {
-	1: u'\u2680',
-	2: u'\u2681',
-	3: u'\u2682',
-	4: u'\u2683',
-	5: u'\u2684',
-	6: u'\u2685'
-}
+dice_sides = {1: 'https://upload.wikimedia.org/wikipedia/commons/6/62/Kismet-Ace.png',
```
Comment (swapagarwal) [inline modules/src/dice.py]: We prefer using images from http://fa2png.io/ wherever possible. They are quite elegant! 😄 
See http://fa2png.io/r/foundation-icon-fonts/die-one/?color=e74c3c&margin=0&size=256
Clicking on generate button yields http://fa2png.io/media/icons/foundation-icon-fonts/2015-02-16/die-one/256/0/e74c3c_none.png (see download button link)

**Comment by abrahamalbert18** [comment]
@swapagarwal  Please check the new update. It uses images from the website " http://fa2png.io/ ". 
I used Attachment template this time.

**Comment by swapagarwal** [comment]
Great work. 🎉 
Thanks @abrahamalbert18! 👍 
Dice module looks really rad now. 😄

**Comment by abrahamalbert18** [comment]
Thank you 😊.

**Pull request merged**